test/webgl-utils/custom-parameter-settings.js

```diff
@@ -140,66 +140,61 @@ export const GL_PARAMETER_SETTINGS_TWO = {
   [GL.UNPACK_SKIP_IMAGES]: 32
 };
 
-// Note: These settings are same as GL_PARAMETER_SETTINGS_TWO
-// When changing change objects.
-export const GL_PARAMETER_SETTINGS_TWO_ENUM_FUNCTION = {
```

Comment:
I moved into `test/webgl`, but still keeping these maps in same file, all three maps are used to set state and also retrieve and verify, didn't see a reason to split, happy to change if you think otherwise.